torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -102,7 +102,8 @@ function HudRow({
           <a
             href={`https://github.com/${params.repoOwner}/${params.repoName}/pull/${rowData.prNum}`}
           >
-            #{rowData.prNum}
+            {rowData.isForcedMerge && (<mark>#{rowData.prNum}</mark>)}
```

Comment:
Thoughts on using something else to mark the force merges? So much yellow highlight is hard on the eyes 😅

Reply:
That's kinda drives the point - maybe there should not be so many force merges?

Reply:
But yeah, maybe something a bit less bright?

Reply:
Yeah, I can switch to something less bright.  I tried to add a symbol or a text message saying that these are forced merge PR, but it makes the columns look ugly because there are so many of them cluttering the view.  So I decided to just highlight them